azure_pipeline.yaml

```diff
@@ -6,6 +6,8 @@ trigger:
   branches:
     include:
     - master
+  pr:
+    - master
```

Comment:
shouldn't this be the default?
in ops-resource-groups I removed the trigger customisation entirely?
https://github.com/hmcts/ops-resource-groups/pull/233

and it still did nothing, but i guess might need a new PR

Reply:
Not sure how it worked so far. It appears that this is required going forward. I asked if they can ask product group about changes to this but support said they probably won't get any useful answer.